src/components/Editor/Tabs.css

```diff
@@ -102,6 +102,15 @@
   background: var(--theme-highlight-blue);
 }
 
+.source-tab img.react {
+  mask: url(/images/react.svg) no-repeat;
+  mask-size: 100%;
+  padding-top: 12px;
+  height: 14px;
+  width: 14px;
+  background: var(--theme-highlight-blue);
```

Comment:
dark mode might benefit from a different color... something darker...

Reply:
![image](https://user-images.githubusercontent.com/9325039/32984623-ae178f76-cc6f-11e7-9931-8895cd469c2d.png)
![image](https://user-images.githubusercontent.com/9325039/32984633-c915a736-cc6f-11e7-9c60-9d18ff47142e.png)

Reply:
This is using bluegrey, thoughts?

Reply:
i like it